src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.js

```diff
@@ -1,125 +1,129 @@
-            angular.copy(notification, blade.notification);
-            if (notification.errorCount > 0) {
-                bladeNavigationService.setError('Import error', blade);
+    .controller('platformWebApp.exportImport.importMainController', ['$scope', 'platformWebApp.bladeNavigationService', 'platformWebApp.exportImport.resource', 'FileUploader', function ($scope, bladeNavigationService, exportImportResourse, FileUploader) {
+        var blade = $scope.blade;
+        blade.updatePermission = 'platform:exportImport:import';
+        blade.headIcon = 'fa fa-download';
+        blade.title = 'platform.blades.import-main.title';
+        blade.isLoading = false;
+        $scope.importRequest = {};
+        var origManifest;
+
+        $scope.$on("new-notification-event", function (event, notification) {
+            if (notification.jobId && notification.finished) {
+                blade.toolbarCommands.splice(0, 1, commandClose);
             }
+            if (blade.notification && notification.id == blade.notification.id) {
+                angular.copy(notification, blade.notification);
+                if (notification.errorCount > 0) {
+                    bladeNavigationService.setError('Import error', blade);
+                }
+            }
+        });
+
+        $scope.canStartProcess = function () {
+            return blade.hasUpdatePermission() && (_.any($scope.importRequest.modules) || $scope.importRequest.handleSecurity || $scope.importRequest.handleSettings || $scope.importRequest.handleBinaryData);
         }
-    });
-
-    $scope.canStartProcess = function () {
-        return blade.hasUpdatePermission() && (_.any($scope.importRequest.modules) || $scope.importRequest.handleSecurity || $scope.importRequest.handleSettings || $scope.importRequest.handleBinaryData);
-    }
-
-    $scope.startProcess = function () {
-        blade.isLoading = true;
-        exportImportResourse.runImport($scope.importRequest,
-            function (data) { blade.notification = data; blade.isLoading = false; },
-            function (error) { bladeNavigationService.setError('Error ' + error.status, blade); });
-
-        blade.toolbarCommands.splice(0, 2, commandCancel);
-    }
-
-    var commandCancel = {
-        name: 'platform.commands.cancel',
-        icon: 'fa fa-times',
-        canExecuteMethod: function () {
-            return blade.notification && !blade.notification.finished;
-        },
-        executeMethod: function () {
-            exportImportResourse.taskCancel({ jobId: blade.notification.jobId }, null, function (data) {
-            });
+
+        $scope.startProcess = function () {
+            blade.isLoading = true;
+            exportImportResourse.runImport($scope.importRequest,
+                function (data) { blade.notification = data; blade.isLoading = false; },
+                function (error) { bladeNavigationService.setError('Error ' + error.status, blade); });
```

Comment:
this error callback is duplicating the default functin and should be removed

Reply:
removed